CMakeLists.txt

```diff
@@ -78,6 +80,12 @@ if (LEGACY_BUILD)
     if (DISABLE_INTERNAL_IMDSV1_CALLS)
         add_definitions(-DDISABLE_IMDSV1)
     endif ()
+    if (USE_TLS_V2)
+        add_definitions(-DENFORCE_TLS_V2)
```

Comment:
hate to be that guy
but there is no TLS V2 or V3
There is V1_2 or V1_3 :-(

Reply:
hahaha yeah can change